docs/Local Development Install.md

```diff
@@ -83,43 +83,27 @@ Let's begin local development install!
 
     ![Docker Desktop](images/dev/3.png "Docker install")
   </details>
-- ### Running containers
 
-  with docker running open a command terminal
 
-      docker pull postgres:16.1
-    
-  this will download the postgres image, then you need to configure postgres
-
-      docker run -d --name dcmfPostgres -p 5432:5432 -e POSTGRES_PASSWORD=dcm -e POSTGRES_USER=dcm postgres
-
-  choose your own credentials of course, please **do not** use credentails shown on the guide.
-  now connect to postgres DB either through your IDE or PGADMIN4
-
-  ![Docker desktop postgres container](images/dev/4.png "Docker postgres")
-
-  ![Postgres connection](images/dev/5.png "Docker postgres connection")
-
-- ### Running project first time for init configs
-  execute all flyaway scripts on the postgres dcm schema.
-  and create a keycloak schema (for later use by keycloak)
-  . Run the project. 
-  and
-  when you see "Completed initialization in 1ms" stop the project. 
-
-- ### Running Keycloak
-  open a new cmd file and run 
-
-      docker run -p 8888:8080 -e KEYCLOAK_ADMIN=YourAdminName -e KEYCLOAK_ADMIN_PASSWORD=YourAdminPassword quay.io/keycloak/keycloak:23.0.6 start-dev
+- ### Running containers  
 
-  Next go to docker and follow the link on the keycloak container. 
-
-  Login with admin credentials, under realms, click create new realm
-  import the realm-export.json on the keycloak folder.
+  This project includes integrated support for Spring Boot Docker Compose, facilitating the setup of Keycloak and PostgreSQL without manual intervention.
+  
+  You won't need to manually start containers or configure settings. The application will reference the `compose.yaml` file located at the root of the project.
+  
+  Feel free to adjust configurations (such as ports and credentials) in the `compose.yaml` file to suit your requirements.
+  
+  When the application starts, it automatically creates a PostgreSQL container with the provided environment credentials. Additionally, a new database for Keycloak is set up.
 
-  ![Postgres connection](images/dev/5.png "Docker postgres connection")
+  On startup, the application also creates a Keycloak container based on the configurations in the Compose file. Initial configurations, including the creation of realms, clients, and users, are performed using the `dcm_realm.json` file.  
+  
+  ![Docker desktop running containers](images/dev/11.png)
```

Comment:
Prefer not to add docker desktop reference. See #68 .  But in this case, it is only representation.